serving/src/main/java/ai/djl/serving/http/KServeRequestHandler.java

```diff
@@ -174,4 +392,124 @@ private void onException(Exception ex, ChannelHandlerContext ctx) {
 
         NettyUtils.sendJsonResponse(ctx, content, status);
     }
+
+    void onModelException(Throwable t, ChannelHandlerContext ctx) {
+        HttpResponseStatus status;
+        if (t instanceof TranslateException) {
+            status = HttpResponseStatus.BAD_REQUEST;
+        } else if (t instanceof WlmException) {
+            logger.warn(t.getMessage(), t);
+            status = HttpResponseStatus.SERVICE_UNAVAILABLE;
+        } else {
+            logger.warn("Unexpected error", t);
+            status = HttpResponseStatus.INTERNAL_SERVER_ERROR;
+        }
+        if (ctx != null) {
+            NettyUtils.sendError(ctx, status, t);
+        }
+    }
+}
```

Comment:
Merge this method with `OnException` method. Since the error format is same for all the APIs in v2.